### PR TITLE
router-advert: T6681: add new option "no-send-interval"

### PR DESCRIPTION
**Pull request by c-po**
## Change Summary

Improve current documentation and add new CLI command.

## Related Task(s)

* https://vyos.dev/T6681

## Related PR(s)

* https://github.com/vyos/vyos-1x/pull/4022

## Backport

sagitta

## Checklist:

- [x] I have read the [**CONTRIBUTING**](https://github.com/vyos/vyos-documentation/blob/current/CONTRIBUTING.md) document

### CODE REVIEW

**Comment by rebortg** [comment]
@mergifyio backport sagitta

**Review by rebortg — APPROVED** [review]

File: docs/configuration/service/router-advert.rst [hunk 1]
```diff
@@ -102,6 +102,13 @@ To disable advertisements without deleting the configuration:
 
 .. cfgcmd:: set service router-advert interface <interface> no-send-advert
 
+   If set, the router will no longer sends periodic router advertisements and not
```
Comment (dmbaturin) [inline docs/configuration/service/router-advert.rst]:
```suggestion
   If set, the router will no longer send periodic router advertisements and will not
```

File: docs/configuration/service/router-advert.rst [hunk 2]
```diff
@@ -102,6 +102,13 @@ To disable advertisements without deleting the configuration:
...
+   responds to router solicitations.
```
Comment (dmbaturin) [inline docs/configuration/service/router-advert.rst]:
```suggestion
   respond to router solicitations.
```

Verb tense typos.

**Review by dmbaturin — CHANGES_REQUESTED** [review]

**Comment by dmbaturin** [comment]
@mergifyio backport circinus

**Pull request merged**